tests/system/fim/common_tasks/verify_alerts_json.py

```diff
@@ -105,16 +105,16 @@ def main():
                             help="Output path for missing files alerts.",
                             default="debug_missing_file_alerts.log")
 
-        parser.add_argument("-t", "--timeout", type=int, required=False, dest='time_gap',
```

Comment:
to avoid misunderstanding

Reply:
Parameters are standardized across scripts, so I think we should keep the `-t` parameter, at least for now. Anyway the description can be improved, maybe `--time_sleep` would be better.

Reply:
Oh, right! But that parameter is `-s` in `verify_alerts_elasticsearch`

Reply:
You are right, I believed we had it as `-t` in `verify_alerts_elasticsearch`. Let's keep the `-s` option.